moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -73,7 +73,6 @@ const useStyles = makeStyles((theme) => ({
     width: "100%",
     color: theme.palette.text.secondary,
     fontSize: ".8rem",
-    margin: "8px 0",
```

Comment:
removed this to address John's request about the spacing on the summary page vs funding page